htdocs/class/xoopsform/renderer/XoopsFormRendererBootstrap3.php

```
@@ -285,146 +282,35 @@ public function renderFormDhtmlTextArea(XoopsFormDhtmlTextArea $element)
-
-        $fontStr .= "&nbsp;{$styleStr}&nbsp;{$alignStr}&nbsp;\n";
-
-        $maxlength = $element->configs['maxlength'] ?? 0;
-        $fontStr .= "<button type='button' class='btn btn-default btn-sm' onclick=\"XoopsCheckLength('"
-                    . $element->getName() . "', '" . $maxlength . "', '"
-            . _XOOPS_FORM_ALT_LENGTH . "', '" . _XOOPS_FORM_ALT_LENGTH_MAX . "');\" title='"
-            . _XOOPS_FORM_ALT_CHECKLENGTH . "'><span class='fa-solid fa-square-check' aria-hidden='true'></span></button>";
-        $fontStr .= "</div></div>";
-
-        return $fontStr;
+        $toolbar = new \XoopsDhtmlToolbar();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '311', column '24'). (undefined)

(MissingImport)